src/main/resources/test_files/runtime-spark/DataFrames/GroupByClauseType2.jq

```diff
@@ -0,0 +1,4 @@
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:29:" :)
+for $x in parallelize("foo", "bar")
+group by $y as string := string-length($x)
+return $y
```

Comment:
This file should be named under the name 'GroupByClauseTypeCheckError'

Reply:
DONE